### DISCUSSION TITLE
How to calculate the thin wire structure with openEMS？

### DISCUSSION THREAD

**Discussion by Jaraxxxxus** [thliebig/openEMS-Project#98] [Q&A Usage]
In the calculation of large targets ( such as lightning towers or transmission lines ), the radius of overhead lines and towers is much smaller than the grid size, which will lead to inaccurate calculation results. Some researchers have reproduced the real results by modifying the permittivity and magnetic permeability of the cells around the thin wire. Can openEMS calculate thin wire at present？

**Comment by thliebig** [comment]
I think you can modify/set the relative background permittivity and permeability. But you would need to set it lower than 1 I guess?
Maybe openEMS does not allow that currently? Maybe you can check that? 
If you then "re-normalize" everything etc. I do not see why this should not work in theory?

**Reply by Jaraxxxxus** [reply]
Thank you. According to the thin wire representation proposed by Noda, I have to force the electric field intensity on the thin wire to be 0, and modify the relative permittivity and permeability of the four cells around the thin wire. But I just started to learn FDTD, so I can hardly modify the content of the FDTD computing engine. Therefore, I choose to modify the relative permittivity and permeability of the cells around the thin wire by function AddMaterial and force the electric field component on the wire to be 0 by function AddExcitation. If it doesn 't work, how should I modify the contents of the computing engine.
Regards！

**Reply by thliebig** [reply]
Why modify it around the wire??
If you use the AddCurve or Curve/Wire primitive in openEMS you already get what you are looking for --> the wire is represented as FDTD edges with E=0 only. 
Why you additionally would want to set the permittivity and permeability around the wire is unclear, I do not see any benefit from that...
